src/crud.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-use-before-define */
+
+import Task from './task.js';
+import {
+  updateToDoList, updateLocalStorage, display, editToDoList, deleteToDo, filterToDo,
+} from './toDoList.js';
+import status from './status.js';
+
+const crud = (() => {
+  let toDoList = [];
+
+  const addNewTask = (task) => {
+    toDoList = updateToDoList(toDoList);
+    toDoList.push(new Task(task.value, false, toDoList.length));
+    task.value = '';
+    updateLocalStorage(toDoList);
+    display(toDoList);
+  };
```

Comment:
- [ ] By default, new tasks should have the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). You ought to increase the index of the `toDoList.length` by 1 so that it can have this functionality as from the requirements. Kindly check out this [video](https://www.loom.com/share/188377746fcb4e7289ee248f5023350f) of your app behavior and make corrections.